__tests__/lib/nimbusRecipe.test.ts

```diff
@@ -61,6 +110,28 @@ describe('NimbusRecipe', () => {
         template: "testTemplate"
       })
     })
+
+    it('returns a specialized BranchInfo object if the recipe is from about:welcome', () => {
+      const nimbusRecipe = new NimbusRecipe(AW_RECIPE)
+
+      const branch = AW_RECIPE.branches[1]
+
+      const branchInfo = nimbusRecipe.getBranchInfo(branch)
+
+      // XXX getBranchInfo is actually going to return a previewLink, which
+      // makes this test kind of brittle. We could refactor this to no longer 
+      // use deepEqual and check for the existence of object properties instead.
```

Comment:
yeah that's basically it. Any change to the shape of NimbusRecipe or getBranchInfo will break the test, since it's testing for the exact encoded previewLink. It's probably not that big of a deal as long as we know about it, but we could refactor if it starts to get annoying.

Reply:
Yeah, let's see if it turns out to be worth tweaking.